torchvision/ops/misc.py

```diff
@@ -132,7 +132,7 @@ def _output_size(dim):
 
 
 # This is not in nn
-class FrozenBatchNorm2d(torch.jit.ScriptModule):
+class FrozenBatchNorm2d(torch.nn.Module):
```

Comment:
@lara-hdr This is fine (and I proposed this initially).

@eellison is this expected that scriptmodules do not work with tracing? You mentioned that we had a `@torch.jit.script_method` annotation in forward, but that is I believe the way to go if we have a `ScriptModule`?

Reply:
`ScriptModule` is sort of a deprecated api at this point; replacing it with `torch.nn.Module` and removing `script_method` is good. 

`ScriptModule` does works with tracing; any function that has a script annotation gets compiled and inlined into the trace.

Reply:
I agree, but given that https://github.com/pytorch/pytorch/pull/22328 is not going to be merged anymore, there is no way recommended way fusing single modules with torchscript anymore?